README.md

```diff
@@ -1,14 +1,32 @@
 # GovWifi Logging API
 
-Session logging for successful authentications
+The GovWifi frontend uses this API to record each session with successful authentications.
```

Comment:
It records unsuccessful authentications as well.

"The GovWifi frontend uses this API to record each authentication request.  It is stored in a database and this data is used for reporting and debugging"